src/controllers/api/notes.js

```diff
@@ -0,0 +1,59 @@
+const { getDataFromFile, writeDataToFile } = require("../../utils");
+const { v4: uuidv4 } = require("uuid");
+
+const validKeys = ["title", "text"];
```

Comment:
Since this is only required inside the `createNotes` function, let's not pollute global scope by declaring it here. Best to declare it directly in the function in which it is used.